src/test/java/io/appium/java_client/android/AndroidDriverTest.java

```diff
@@ -140,4 +141,33 @@
         assertNotNull(driver.getDisplayDensity());
         assertNotEquals(0, driver.getSystemBars().size());
     }
+
+    @Test public void getSupportedPerformanceDataTypesTest() {
+        driver.startActivity("io.appium.android.apis", ".ApiDemos");
+
+        List<String> supportedPerformanceDataTypes = driver.getSupportedPerformanceDataTypes();
+        assert(supportedPerformanceDataTypes.size() == 4);
+
+    }
+
+    @Test public void getPerformanceDataTest() {
+        driver.startActivity("io.appium.android.apis", ".ApiDemos");
+
+        List<String> supportedPerformanceDataTypes = driver.getSupportedPerformanceDataTypes();
+
+        for(int i = 0 ; i < supportedPerformanceDataTypes.size() ;  ++ i){
```

Comment:
i applied it. 
but I think there is no reason to check the value of the table, because the values could be changed in appium-android-driver. 
I have a plan to add some other values for example, heaphistogram....